people.md

```diff
@@ -0,0 +1,47 @@
+# Steering Council and Institutional Partners
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
+- Brian Granger
+- Damian Avila
+- Fernando Perez
+- Jason Grout
+- Jessica Hamrick
+- Jonathan Frederic
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
```

Comment:
Okay, how about we use this list of names for now, which is hardcoded in the governance document right now.  Then we open a new PR for updating the list of names.

This PR is about "splitting the repo".  Then we can update the "name repo" in a separate PR.